contracts/test/emergencyTest.js

```diff
@@ -86,7 +86,22 @@ contract('PoolDeposits', accounts => {
     await time.increase(time.duration.days(2));
 
     await poolDeposits.voteEmergency({ from: accounts[2] });
-    await poolDeposits.declareStateOfEmergency({ from: accounts[2] });
+
```

Comment:
This is failing. Unsure of how to best test the timestamp emitted from the event. This was the answer online, but its out by a small margin/

Reply:
Hmm, are you moving time in any of the other tests? That would break this.

Also, you can get the block timestamp and check that it the same as the timestamp, that is best, because internal blockchain time vs external time can always change by progressing time in another test.

Reply:
This is fixed please ignore. PR has been updated

Reply:
This is fixed please ignore. Old comment. I now do get the latest block timestamp.